data/README.md

```diff
@@ -15,7 +15,8 @@ En cas de problème vous pouvez redémarrer de zéro en faisant d'abord un `make
 ## Autres commandes :
 
 * `make image` : pour construire l'image docker choisie
-* `make import_agribalyse` : pour importer la base dans Brightway. Assurez-vous d'avoir le fichier `agribalyse3_no_param.CSV.zip` dans le dossier `food/import_agb/`
+* `make import_agribalyse` : pour importer Agribalyse 3.0 dans Brightway. Assurez-vous d'avoir le fichier `agribalyse3_no_param.CSV.zip` dans le dossier `food/import_agb/`
+* `make import_ei391_agb311` : pour importer Ecoinvent 3.9.1 et Agribaylyse 3.1.1. Assurez-vous d'avoir les bons fichiers dans dans le dossier `data/`
 * `make export_builder` : pour exporter les json pour le builder
 * `make json` : lance toutes les commandes précédentes dans l'ordre
 * `make export_ciqual` : pour exporter les json des produits ciqual. Assurez-vous d'avoir le fichier `Agribalyse_Synthese.csv` dans le dossier `food/`
```

Comment:
This one fails this way:

```
Using environment variable BRIGHTWAY2_DIR for data directory:
/home/jovyan/data
Get ciqual codes from ../Agribalyse_Synthese.csv
Open the Agribalyse 3.0 brightway database
Biosphere database already present!!! No setup is needed
Search for the ciqual products in the brightway database
Loaded 0 productscts
Building product tree
100%
Export de 0 produits vers ../../../public/data/food/products.json
Traceback (most recent call last):
  File "/home/jovyan/ecobalyse/data/food/export_agb/export_ciqual.py", line 315, in <module>
    random_process = next(iter(processes))
StopIteration
make: *** [export_ciqual] Error 1
```

Reply:
Also note that the `make notebook` and `make bash` commands, documented in the README, don't seem to exist anymore.